lib/LINE/Bot/API/Client/Furl.pm

```diff
@@ -132,6 +132,30 @@ sub post_image {
     $ret;
 }
 
+sub put {
+    my($self, $url, $data) = @_;
+
+    my $json = $JSON->encode($data);
+    my($res_minor_version, $res_status, $res_msg, $res_headers, $res_content) = $self->{furl}->put(
+        $url,
+        [
+            $self->credentials,
+            'Content-Type'   => 'application/json',
+            'Content-Length' => length($json),
+        ],
+        $json,
+    );
+
+    unless ($res_content && $res_content =~ /^\{.*\}$/) {
```

Comment:
Not the scope fo this PR, but I think this needs to be brought up.

I just noticed that this regexp, and other copies of the same regexp, might be errornous -- `.` character does not match newline `\n` without modifier `/s` in place.

See: https://perldoc.pl/perlre#Metacharacters

If any response contains prettified JSON text, sanity checks like this becomes bogus.

We can probably refactor them  as `eval { $JSON->decode(...) ` and do proper error-handling instead.